### PR TITLE
Add post on groovy testing

### PR DESCRIPTION
**Pull request by bdemers**
This blog post has the following features:

- [x] A GitHub Repository with a polished README
- [x] A title that's approved by Dev Marketing
- [ ] A URL that's approved by Dev Marketing
- [x] Rendered locally and confirmed that no Markdown typos exist

URL: /blog/2019/08/21/make-java-tests-groovy-again

### CODE REVIEW

File: _source/_posts/2019-08-21-make-java-tests-groovy-again.md [hunk 1]
```diff
@@ -0,0 +1,372 @@
+---
+layout: blog_post
+title: Make Java Tests Groovy Again With Hamcrest
+description: 'A tutorial on testing Java with Groovy'
+tags: spring, oauth, oauth 2.0, java, spring boot, testing, hamcrest, groovy
```
Comment (mraible) [inline _source/_posts/2019-08-21-make-java-tests-groovy-again.md]: This should be an array
```suggestion
tags: ]spring, oauth, oauth 2.0, java, spring boot, testing, hamcrest, groovy]
```

File: _source/_posts/2019-08-21-make-java-tests-groovy-again.md [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+public String sayHello(String name) {
+    return sayHello(name, "How are you?");
+}
+
+public String sayHello(String name, String greeting) {
+    System.out.println("Hello " + name + ", " greeting);
+}
+```
+
+The Groovy block is faster to write more concise.
```
Comment (mraible) [inline _source/_posts/2019-08-21-make-java-tests-groovy-again.md]: and more concise?

File: _source/_posts/2019-08-21-make-java-tests-groovy-again.md [hunk 3]
```diff
@@ -0,0 +1,372 @@
+---
+layout: blog_post
+title: Make Java Tests Groovy Again With Hamcrest
+description: 'A tutorial on testing Java with Groovy'
```
Comment (mraible) [inline _source/_posts/2019-08-21-make-java-tests-groovy-again.md]:
```suggestion
description: 'A tutorial on testing Java code with Groovy.'
```

File: _source/_posts/2019-08-21-make-java-tests-groovy-again.md [hunk 4]
```diff
@@ -0,0 +1,372 @@
...
+Change the Login redirect URIs to `http://localhost:8080/login/oauth2/code/okta`.
+
+Click **Done**.
+
+Take note of the Client ID and Client Secret on the next page and update the  `src/main/resources/application.yml`.
+
+```yaml
+okta:
+  oauth2:
+    issuer: {yourOktaDomain}/oauth2/default
```
Comment (mraible) [inline _source/_posts/2019-08-21-make-java-tests-groovy-again.md]:
```suggestion
    issuer: https://{yourOktaDomain}/oauth2/default
```

Comment (bdemers) [reply _source/_posts/2019-08-21-make-java-tests-groovy-again.md]: ugg, I cannot believe I missed that one, i couldn't wait until we switched all the refs to `https://{yourOktaDomain}`

**Pull request merged**